src/pages/ProAmmPool/PositionListItem.tsx

```diff
@@ -185,12 +258,35 @@ function PositionListItem({
       : undefined
 
   const usd =
-    parseFloat(position?.amount0.toExact() || '0') * prices[0] +
-    parseFloat(position?.amount1.toExact() || '0') * prices[1]
+    parseFloat(position?.amount0.toExact() || '0') * prices[token0?.wrapped.address || ''] +
```

Comment:
`?.wrapped?.address`

Reply:
if token is exist, it alway has `wrapped`. dont need to add `?` after that